### ISSUE TITLE
[BUG] <title>The clear_device_cache function cannot be found in accelerate.utils.memory.

### ISSUE THREAD

**Issue by zhiniaodawwang** [OpenBMB/MiniCPM-V#926]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [x] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [x] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

While fine-tuning minicpm2_6 in Lora, I downloaded the required software version according to requirement.txt. However, the 0.30.1 version of accelerate does not include clear_device_cache. What version of the software should I download?

(MiniCPM-V) ➜  finetune git:(main) ✗ pip list
Package                  Version
------------------------ --------------
accelerate               0.30.1
addict                   2.4.0
aiofiles                 23.2.1
annotated-types          0.7.0
anyio                    4.9.0
attrs                    25.3.0
certifi                  2025.4.26
charset-normalizer       3.4.2
click                    8.2.1
colorama                 0.4.6
contourpy                1.3.2
cycler                   0.12.1
decord                   0.6.0
deepspeed                0.15.4+unknown
editdistance             0.6.2
einops                   0.7.0
et_xmlfile               2.0.0
exceptiongroup           1.3.0
fairscale                0.4.0
fastapi                  0.115.12
ffmpy                    0.6.0
filelock                 3.18.0
fonttools                4.58.1
fsspec                   2025.5.1
gradio                   4.41.0
gradio_client            1.3.0
h11                      0.16.0
hf-xet                   1.1.3
hjson                    3.1.0
httpcore                 1.0.9
httpx                    0.28.1
huggingface-hub          0.32.4
idna                     3.10
importlib_resources      6.5.2
Jinja2                   3.1.6
joblib                   1.5.1
jsonlines                4.0.0
kiwisolver               1.4.8
lxml                     5.4.0
markdown-it-py           3.0.0
markdown2                2.4.10
MarkupSafe               2.1.5
matplotlib               3.7.4
mdurl                    0.1.2
modelscope_studio        0.4.0.9
more-itertools           10.1.0
mpmath                   1.3.0
msgpack                  1.1.0
networkx                 3.4.2
ninja                    1.11.1.4
nltk                     3.8.1
numpy                    1.24.4
nvidia-cublas-cu12       12.1.3.1
nvidia-cuda-cupti-cu12   12.1.105
nvidia-cuda-nvrtc-cu12   12.1.105
nvidia-cuda-runtime-cu12 12.1.105
nvidia-cudnn-cu12        8.9.2.26
nvidia-cufft-cu12        11.0.2.54
nvidia-curand-cu12       10.3.2.106
nvidia-cusolver-cu12     11.4.5.107
nvidia-cusparse-cu12     12.1.0.106
nvidia-ml-py             12.575.51
nvidia-nccl-cu12         2.18.1
nvidia-nvjitlink-cu12    12.9.41
nvidia-nvtx-cu12         12.1.105
opencv-python-headless   4.5.5.64
openpyxl                 3.1.2
orjson                   3.10.18
packaging                23.2
pandas                   2.2.3
peft                     0.15.2
Pillow                   10.1.0
pip                      25.1
portalocker              3.1.1
protobuf                 4.25.0
psutil                   7.0.0
py-cpuinfo               9.0.0
pydantic                 2.9.2
pydantic_core            2.23.4
pydub                    0.25.1
Pygments                 2.19.1
pyparsing                3.2.3
python-dateutil          2.9.0.post0
python-multipart         0.0.20
pytz                     2025.2
PyYAML                   6.0.2
regex                    2024.11.6
requests                 2.32.3
rich                     14.0.0
ruff                     0.11.12
sacrebleu                2.3.2
safetensors              0.5.3
seaborn                  0.13.0
semantic-version         2.10.0
sentencepiece            0.1.99
setuptools               78.1.1
shellingham              1.5.4
shortuuid                1.0.11
six                      1.17.0
sniffio                  1.3.1
socksio                  1.0.0
starlette                0.46.2
sympy                    1.14.0
tabulate                 0.9.0
timm                     0.9.10
tokenizers               0.19.1
tomlkit                  0.12.0
torch                    2.1.2
torchvision              0.16.2
tqdm                     4.66.1
transformers             4.40.0
triton                   2.1.0
typer                    0.16.0
typing_extensions        4.8.0
tzdata                   2025.2
urllib3                  2.4.0
uvicorn                  0.24.0.post1
websockets               12.0
wheel                    0.45.1

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```
